core/base-shaper.lua

```diff
@@ -59,14 +59,21 @@ SILE.shapers.base = pl.class({
       return shapespace(width and width.length or items[1].width)
     end,
 
-    measureChar = function (self, char)
+    measureChar = function (self, char, fallible)
```

Comment:
This was a good idea and I see why you took it this direction, but I do want to think whether it might not be better to leave this function alone and just call it with a `pcall()` wrapper for the rare cases we want to let it fail. I need more sleep to think through those ramifications though...

Reply:
Or scoped to the font?

Reply:
I refactored this going back to throwing an error but using `pcall()` to wrap the one case where we want to allow that and output the warning there. The overall logic should be the same.